config/software/openssl.rb

```diff
@@ -44,6 +44,8 @@
   source url: "https://www.openssl.org/source/openssl-#{version}.tar.gz", extract: :lax_tar
 end
 
+version("3.0.5")   { source sha256: "aa7d8d9bef71ad6525c55ba11e5f4397889ce49c2c9349dcea6d3e4f0b024a7a" }
+version("3.0.4")   { source sha256: "2831843e9a668a0ab478e7020ad63d2d65e51f72977472dc73efcefbafc0c00f" }
```

Comment:
Since this is additive, it could be a change of its own.

Reply:
Happily this was added by someone else and merged already. I can kill this off